packages/editor/src/pages/Editor/store/configureStore.ts

```diff
@@ -13,13 +13,15 @@ const addLoggingToDispatch = store => {
     return rawDispatch;
   }
   return action => {
-    console.group(action.type);
-    console.log('%c prev state', 'color: gray', store.getState());
-    console.log('%c action', 'color: blue', action);
-    const returnValue = rawDispatch(action);
-    console.log('%c next state', 'color: green', store.getState());
-    console.groupEnd();
-    return returnValue;
+    if (action && action.type) {
```

Comment:
In what instance would there be an action dispatched that is null? Or what action would not have a type?

Reply:
That's a very good question.  However, it somehow happened to me once.  So I looked where the `undefined` was being de-referenced and just added this extra check in case.  It feels like we shouldn't crash an app for our own logging issues...

Reply:
I feel like we should (esp since it is only on localhost) if someone is dispatching a null action, otherwise, it's just putting a bandaid over the actual problem